loony_dev/web/static/app.css

```
@@ -2,146 +2,140 @@
    No build step: plain CSS with custom-property design tokens. */
 
 /* ---- Design tokens --------------------------------------------------- */
+/* The token + type layer IS the L Space design system, vendored verbatim as
+   /static/design/colors_and_type.css and loaded before this file (see
+   index.html). That file owns the --gray-*/--ink neutral ramp, the
```

Stylelint (17.13.0)

[error] Unknown word neutral (CssSyntaxError)

(CssSyntaxError)